packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -1,228 +1,59 @@
-                                onChange={this.handleInputChange}
-                                type="text"
-                                state={verifyAddressError ? 'error' : undefined}
-                                bottomText={verifyAddressError ? verifyAddressError.message : null}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_MESSAGE)}
-                                name="verifyMessage"
-                                value={verifyMessage}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <Row>
-                            <Textarea
-                                topLabel={intl.formatMessage(messages.TR_SIGNATURE)}
-                                name="verifySignature"
-                                value={verifySignature}
-                                onChange={this.handleInputChange}
-                                rows={4}
-                                maxRows={4}
-                                maxLength={255}
-                            />
-                        </Row>
-                        <RowButtons>
-                            <StyledButton onClick={signVerifyActions.clearVerify}>
-                                <Translation id="TR_CLEAR" />
-                            </StyledButton>
-                            <StyledButton
-                                // isDisabled={!!verifyAddressError || !device.connected}
-                                isDisabled
-                                onClick={() => {
-                                    if (errors.length <= 0) {
-                                        signVerifyActions.verify(
-                                            verifyAddress,
-                                            verifyMessage,
-                                            verifySignature,
-                                        );
-                                    }
-                                }}
-                            >
-                                <Translation id="TR_VERIFY" />
-                            </StyledButton>
-                        </RowButtons>
-                    </Verify>
-                </Wrapper>
-            </WalletLayout>
-        );
-    }
-}
+    return (
+        <WalletLayout title="Sign & Verify" account={selectedAccount}>
+            <SignVerifyContainer>
+                <SignMessage
+                    notificationActions={notificationActions}
+                    account={selectedAccount.account}
+                    isLocked={isLocked}
+                    device={suite.device}
```

Comment:
you can get device object from `useDevice` hook instead of adding `props.suite`

in fact you'll not need `device` here at all if you move back actions to where they belong